### ISSUE TITLE
Testdata error

### ISSUE THREAD

**Issue by Hydaspex** [AutoViML/Auto_TS#69]
Hi gents,
When i run model. Predict a silly error comes up.
It is related to the parameter testdata and returns object of type float has no len() 
Please let me know if you have a quick solution. 
Thanks for your help.

**Comment by AutoViML** [comment]
Hi @Hydaspex 👍 
It is fixed now. Please do the upgrade to latest version by:
pip install auto-ts --upgrade

Please confirm that fixes the problem.
Thanks
Auto_ViML

**Comment by Hydaspex** [comment]
Yes it works now however there seems to be an issue in processing the testdata's  index when the model is ML.
The error message is quoted below:

`redicting using test dataframe as input for ML model
For large datasets: ML predictions will take time since it has to predict each row and use that for future predictions...
    Using given input: pandas dataframe...
    converting testdata to datetime index erroring. Please check input and try again.
ML predictions completed`

Thanks

**Comment by AutoViML** [comment]
Hi @Hydaspex 👍 
Can you please post a Colab notebook or GitHub link for your notebook where you re getting this error. It would help me find out the issue quickly. Can you please give me more input here?

**Comment by Hydaspex** [comment]
Hi @AutoViML,

you can find the notebook at the following repository

https://github.com/Hydaspex/rrt_waiting_times.git

Thanks

**Comment by rsesha** [comment]
Hi @Hydaspex 👍 
I noticed that you have set the time index already. Instead, I commented it out. Just send the dataframe as is and auto-ts will automatically set the time index. Once you make that one line change, it works automatically on train and test. Take a look at the notebook below.
[rtt_model_v2.zip](https://github.com/AutoViML/Auto_TS/files/7780799/rtt_model_v2.zip)

One more thing: upgrade to the latest version and test it:
```pip install auto-ts --upgrade ```

Close this issue if it is fixed.
AutoViML team
